check_filter.py

```diff
@@ -40,14 +40,14 @@ def check_filter(query_file, query_udfs):
     copytree(filter_dir, target_filter_dir)
 
     # build the filter
-    cmd = "cargo build "
+    cmd = "cargo +nightly build "
```

Comment:
This should not be necessary

Reply:
It does not build without that addition for me.  Since it's automatic if you have the nightly toolchain automatically on, and doesn't hurt if you don't, I figured it couldn't hurt.

Reply:
`rustup default nightly  ` should work. If you add these, you have to remove all of them if you switch tool chains at some later point. Otherwise it is just a simple switch at the beginning.

Reply:
Gotcha.  I'll do that